src/components/UI/docs/DocumentNav.tsx

```diff
@@ -28,10 +28,27 @@ export const DocumentNav: FC<Props> = ({ content }) => {
       {parsedHeadings.map((heading, idx) => {
         return (
           <NextLink key={`${idx} ${heading?.title}`} href={`#${heading?.headingId}`}>
-            <Link m={0}>
+            <Link m={0} textDecoration='none !important'>
```

Comment:
link `:hover` underline color [should be different ](https://www.figma.com/file/ekzIgwyeVKLtFSAcnA0Q0D/geth-website?node-id=780%3A8008&t=2m75Yb2GVQIwfYbY-0), also `:pressed` status colors missing

Reply:
> link :hover underline color should be different

I'm not sure I understand... This PR updates the links in the DocumentNav component to use the styling below, which does not have an underline:

<img width="437" alt="image" src="https://user-images.githubusercontent.com/54227730/205728119-d601ed37-e710-487c-9ba9-b6e1ddbd76d2.png">

[Figma board](https://www.figma.com/file/ekzIgwyeVKLtFSAcnA0Q0D/geth-website?node-id=2396%3A19079&t=IM0cQlwLjB4YYIVG-1)

---

> also :pressed (_pressed prop) status colors missing

Which state is this even?

<img width="586" alt="image" src="https://user-images.githubusercontent.com/54227730/205728961-c32b87a0-014e-4cd2-8cde-26c93c5b405f.png">

Shouldn't `_focus` cover this state?

<img width="328" alt="image" src="https://user-images.githubusercontent.com/54227730/205472323-04f8c7ee-1013-49b7-8229-e53eb829023c.png">

Reply:
I think we're mixing the issues, #89 is about links in the docs body (active/pressed state), I don't know when :pressed is triggered, cc @nloureiro

Reply:
So #89 appears to be referring to the DocumentNav component being worked on in this PR, not standard links.

Originally, this component was displaying these as regular _links_ and @minimalsm noted the design system for _link_ styling... but then @nloureiro followed up, clarifying that the hover effect for these links was documented in #73, but he forgot to mention the active and focus states. He displays screenshots of what these two states should look like in #89, which I've added to this PR.

Reply:
@nhsz I just commented on #89... tl;dr, looks like we need to add active and focus states for _both_ the DocumentNav as well as the body links.... For this PR, let's just keep this where it is, and I'll address the links in the body separately.

Reply:
Put up a separate PR #129 to address styling the body links